app/routes/_layout+/calendar.tsx

```diff
@@ -214,6 +239,62 @@ const Calendar = () => {
               eventMouseEnter={handleEventMouseEnter}
               eventMouseLeave={handleEventMouseLeave}
               windowResize={handleWindowResize}
+              eventContent={(args) => {
+                const hoveredBooking = args.event
+                  .extendedProps as CalendarExtendedProps;
+
+                const startTime = format(
+                  new Date(hoveredBooking.start),
+                  "hh:mm"
+                );
+
+                return (
+                  <HoverCard openDelay={0} closeDelay={0}>
+                    <HoverCardTrigger className="inline-block w-full">
+                      {startTime} | {args.event.title}
+                    </HoverCardTrigger>
+
+                    <HoverCardContent
+                      className="pointer-events-none md:w-96"
+                      side="top"
+                    >
+                      <div className="flex w-full items-center gap-x-2 text-xs text-gray-600">
+                        {format(new Date(hoveredBooking.start), DATE_FORMAT)}
```

Comment:
We have a new way of managing this. We have a component called `DateS` that automatically formats the date based on the user's timezone and locale and everything, using the client hints. From now on we have to always use that component.